### ISSUE TITLE
Bad error message when using unsupported FULLTEXT KEY

### ISSUE THREAD

**Issue by avaitla** [dolthub/dolt#2978]
FULLTEXT KEY not supported:

```
CREATE TABLE `another_random_table` (
  `id` int NOT NULL AUTO_INCREMENT,
  `body` text NOT NULL,
  PRIMARY KEY (`id`),
  FULLTEXT KEY `idx_body` (`body`)
) ENGINE=InnoDB AUTO_INCREMENT=223 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
ERROR 1105 (HY000): syntax error at position 135 near 'KEY'
```

**Comment by avaitla** [comment]
This seems pretty minor to me though, but if you wanted to shoot for compatibility something to look at.

**Comment by zachmu** [comment]
We have no near-term plans to support fulltext indexes, but this should parse and give a better error message ("unsupported feature"), as the syntax is correct.

**Comment by timsehn** [comment]
Renamed this one to fix the error message and added #2987 for the feature request.
